main.js

```diff
@@ -0,0 +1,19 @@
+const table = document.getElementById('table');
+
+function sortRows(event) {
+    const rows = table.rows;
+    const dataAttribute = event.target.getAttribute('data-type');
+    const newRowsTable = [];
+    for (let i = 1; i < rows.length; i++) {
+        newRowsTable.push(rows[i]);
+    }
+
+    if (dataAttribute === 'number') {
+        newRowsTable.sort((a, b) => a.cells[0].innerText - b.cells[0].innerText);

```

Comment:
do not hardcode indices, my table has the third row. 😉  Also what if I switch rows?